### PR TITLE
Add ability for skins to have more control over judgement transforms

### LINKED ISSUE #10195: For legacy skins, judgements should use animations that match stable
Re-opening #6420 under this name

**Describe the bug:** 
Legacy skin judgements use different animation effects on stable than they do on lazer. On lazer they pop out from middle regardless of if the judgements are animated or not, on stable they fade in if they are animated and only pop out if not.

**Screenshots or videos showing encountered issue:** 
https://youtu.be/Je50_IcAmFs
lazer is on left, stable on right, using [my skin](https://github.com/LittleEndu/osuSkin/tree/master/u200B)

**osu!lazer version:** 
2020.910.0

### PR DESCRIPTION
**Pull request by peppy**
Closes #10195.

Can be tested with existing test scene (`osu.Game.Rulesets.Osu.Tests.TestSceneDrawableJudgement`) – special skin is animated and should no longer show a scale transform on misses *or* hits. All others should remain as previous.

Ended up implementing this in a non-breaking way (I believe).

### CODE REVIEW

**Review by bdach — APPROVED** [review]
pushed two menial changes, otherwise looks good to me :+1:

File: osu.Game/Rulesets/Judgements/DrawableJudgement.cs [hunk 1]
```diff
@@ -91,33 +109,31 @@ protected override void PrepareForUse()
...
                     break;
 
                 default:
                     ApplyHitAnimations();
                     break;
             }
 
+            if (skinnableJudgement.Drawable is IAnimatableJudgement animatable)
+            {
+                using (BeginAbsoluteSequence(Result.TimeAbsolute))
```
Comment (smoogipoo) [inline osu.Game/Rulesets/Judgements/DrawableJudgement.cs:133]: This won't have any effect because it doesn't have `recurse = true`.

Comment (peppy) [reply osu.Game/Rulesets/Judgements/DrawableJudgement.cs:133]: should be fixed in the follow-up PR already

Comment (smoogipoo) [reply osu.Game/Rulesets/Judgements/DrawableJudgement.cs:133]: Indeed it's removed ':D

Comment (peppy) [reply osu.Game/Rulesets/Judgements/DrawableJudgement.cs:133]: not removed, just much further out in scope.

**Pull request merged**